src/core/src/op/interpolate.cpp

```diff
@@ -147,16 +147,37 @@ static constexpr size_t target_shape_port = 1;
 static constexpr size_t scales_port = 2;
 static constexpr size_t axes_port = 3;
 static constexpr size_t max_num_of_ports = 4;
+size_t get_scales_port(const ov::op::util::InterpolateBase* node) {
+    if (ov::is_type<ov::op::v11::Interpolate>(node)) {
+        return scales_port - 1;
+    }
+    return scales_port;
+}
+
+size_t get_axes_port(const ov::op::util::InterpolateBase* node) {
+    if (ov::is_type<ov::op::v11::Interpolate>(node)) {
+        return axes_port - 1;
+    }
+    return axes_port;
+}
+
+size_t get_max_num_of_ports(const ov::op::util::InterpolateBase* node) {
+    if (ov::is_type<ov::op::v11::Interpolate>(node)) {
+        return max_num_of_ports - 1;
+    }
+    return max_num_of_ports;
+}
```

Comment:
Once these values depend on the op version, should these be hidden as function local variables?

Reply:
Good point, moved to the local scope.